### ISSUE TITLE
FR: Start Tailscale Disconnected

### ISSUE THREAD

**Issue by maptan** [tailscale/tailscale#8585]
### What are you trying to do?

Most of the time I'm inside my network, and tailscale will sometimes prevent local resources from being accessed, or connection to local resources are intermittent. When that happens, after some troubleshooting, I remember to disconnect Tailscale, and everything works. Ideally, I'd like the preference to start disconnected and connect on demand.

### How should we solve this?

- Having a preference to start disconnected/connect on demand. 
- Another possibility would be some kind of parameter on admin panel that would set the specific client to start disconnected.
- Maybe think in some kind of detection based on the network name or detecting some anchor device.

### What is the impact of not solving this?

Some time lost trying to understand why things do not work as they should, and remembering to disconnect.

**Comment by blong112** [comment]
My previous company uses Zscaler and it would automatically disconnect if it is on the company LAN. If I am not wrong, the Zscaler client would scan for a host if you are on a 'trusted network' and disconnect itself (or it disables its service). Not really a biggie but convenient for bigger organizations where some users don't even know how what is the meaning of 'right click'.

**Comment by maptan** [comment]
> My previous company uses Zscaler and it would automatically disconnect if it is on the company LAN. If I am not wrong, the Zscaler client would scan for a host if you are on a 'trusted network' and disconnect itself (or it disables its service). Not really a biggie but convenient for bigger organizations where some users don't even know how what is the meaning of 'right click'.

Yes, I've already setup this in some clients using AlwaysOn VPN. That's pretty common in windows world, [this is an example](https://directaccess.richardhicks.com/2020/03/24/always-on-vpn-trusted-network-detection/)... but it is sometimes glitchy, as Windows messes up network names sometimes. Also, this might not be useful in other OSes, so, maybe an admin panel option would be the best option, imho.

**Comment by DentonGentry** [comment]
Likely duplicate of https://github.com/tailscale/tailscale/issues/983

**Comment by janhurst** [comment]
> Likely duplicate of [#983](https://github.com/tailscale/tailscale/issues/983) Barring further comment, we'll close this as a duplicate during the next regular bug scrub.

I think there is a slightly different ask than [#983](https://github.com/tailscale/tailscale/issues/983)... I would prefer that the GUI auto starts however Tailscale is NOT connected (until I go and ask it to).

My use case is that 95% of the time my users do not need to connect to Tailscale directly, I have subnet routers dealing with my corporate LAN/WAN... however when they go remote we want to connect to the tailnet.

(yes I understand they can just start the GUI...)
